### DISCUSSION TITLE
When rendering over a black backrgound, KaTeX renders the equation twice.

### DISCUSSION THREAD

**Discussion by Uncodeable864** [KaTeX/KaTeX#3731] [Q&A] [answered]
I'm attempting to render some math over a darker background (`#111`), but I find that KaTeX and other libraries render the equation twice. Anybody have any solutions to fix this?

Example for $a=a$:
<img width="109" alt="image" src="https://user-images.githubusercontent.com/49599682/193463184-9bdf0beb-6539-492d-80f9-3d47da70d951.png">?

KaTeX code:

```js
      Katex.renderToString(`\\color{#eee}` + formula, { displayMode }),
```

**Comment by Uncodeable864** [comment, marked as answer]
Turns out, _you need to import the KaTeX stylesheet_. That solved the problem.

**Reply by edemaine** [reply]
Ah, yes. The double rendering you're seeing is the MathML and HTML. With the style sheet, the MathML disappears visually (while hopefully still appearing to screen readers).
